cluster-provision/k8s/1.17/ovsdpdk.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+set -ex
+
+setenforce 0
```

Comment:
Why is that necessary? I understood we were making it possible to leave selinux enabled since 1.17? @jean-edouard WDYT?

Reply:
There is need to share the vhostuser ports created by libvirt inside the EmptyDir volume to be used by OvS ports, by locally mounting it to ``/var/lib/vhost_sockets`` directory (addtional details [here](https://github.com/kubevirt/kubevirt/pull/3208#issuecomment-607765265)) . It had access failures, it requires update in selinux policies, which can be done in order to remove this. This change will be temporary.